lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -228,8 +285,43 @@ rule RootWebHome {
 	msc_p2:XWIKI!Property (
 		name <- 'name',
 		type <- 'String',
-		value <- 'Set of models' --TODO: there is no model set name defined in our process
+		value <- 'Set of models'
+	),
+	--begin documentation
+	msc1:XWIKI!Object (
+		className <- 'LPCode.ModelSetClass',
+		number <- 1,
+		wiki <- 'xwiki',
+		space <- thisModule.getModelSetId(),
+		pageName <- 'WebHome',
+		property <- Sequence{msc1_p0, msc1_p1, msc1_p2, msc1_p3, msc1_p4}->asSequence()
+	),
+	msc1_p0:XWIKI!Property (
+		name <- 'modelid',
+		type <- 'String',
+		value <- ''
+	),
+	msc1_p1:XWIKI!Property (
+		name <- 'id',
+		type <- 'String',
+		value <- 'WebHome'
+	),
+	msc1_p2:XWIKI!Property (
+		name <- 'name',
+		type <- 'String',
+		value <- 'WebHome'
+	),
+	msc1_p3:XWIKI!Property (
+		name <- 'type',
+		type <- 'StaticList',
+		value <- 'OverviewModel'
+	),
+	msc1_p4:XWIKI!Property (
+		name <- 'documentation',
+		type <- 'TextArea',
+		value <- 'WebHome'
```

Comment:
The value should contains the description from the overview model, see [here](https://github.com/MDEGroup/learnpad/blob/VilniusDecisionV1/lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/model/ado4f16a6bb-9318-4908-84a7-c2d135253dc9.xml#L5676).